src/main/java/mate/controller/DeleteCarController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector = Injector
+            .getInstance("mate");
+    private CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long carId = Long.valueOf(req.getParameter("id"));
+        carService.delete(carId);
+        resp.sendRedirect(req.getContextPath() + "/cars/all");
```

Comment:
check your project after refactoring
```suggestion
        resp.sendRedirect(req.getContextPath() + "/cars");
```